Update bio for benhylau (#106)
ournetworks/ournetworks.ca: about.md

Our Networks organizers have hosted civic tech events and community networks workshops, created network literacy materials, and built mesh networking software for the past two years.

> **Benedict Lau**{:#benhylau} is an engineer telling stories of technology practices that bring communities together. He studies distributed protocols and collective governance of digital infrastructures. When he isn't "on emails", he builds passable open source tools and facilitates activities about building peer-to-peer local networks as a way to co-imagine a future equitable web. He is a member of the Hypha Worker Co-operative and core contributor at [Toronto Mesh](https://tomesh.net).

Reviewer: ```suggestion
**Benedict Lau**{:#benhylau} is an engineer who tells stories of technology practices that bring communities together. He studies distributed protocols and collective governance of digital infrastructures. When not "on email", he builds passable open source tools and facilitates activities about peer-to-peer local networks as a way to co-imagine a future equitable web. He is a member of the Hypha Worker Co-operative and a core contributor at [Toronto Mesh](https://tomesh.net).
```

Reviewer: @benhylau -- trying to increase agreement / streamline parallel structure of sentences.
